<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Supprimer_fournisseur</class>
 <widget class="QDialog" name="Supprimer_fournisseur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>470</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_addresse_fournisseur">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gerer_stock">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Gérer le stock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Supprimer_Fournisseur">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>560</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Algerian&quot;;
background-color: rgb(255, 22, 5);
color: rgb(255, 255, 255);border-radius:7px;</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Numero_Telephone">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>390</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>661</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 255, 192);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>19</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 300 17pt &quot;Bahnschrift Light&quot;;border:2px solid  rgb(16, 255, 76);
background-color: rgb(255, 255, 255);border-radius:15px;
color: rgb(86, 171, 17);</string>
    </property>
    <property name="text">
     <string>Veuillez saisir les informations du fournisseur à supprimer</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);font-size:15px;
font: 600 16pt &quot;Segoe UI Variable Text Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Dr.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/WhatsApp Image 2024-04-30 à 20.55.44_6b270705.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Nom_fournisseur">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>270</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="name_user">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
font: 900 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gerer_rupture_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Statistiques</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonProfil">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;border:2px solid black;
background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>20</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 300 17pt &quot;Bahnschrift Light&quot;;</string>
   </property>
   <property name="text">
    <string>Nom de Fournisseur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/OIP (3).jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>16</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Precedent">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>560</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Algerian&quot;;
background-color: rgb(15, 191, 255);
color: rgb(255, 255, 255);border-radius:7px;</string>
   </property>
   <property name="text">
    <string>Précédent</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid blue;border-radius:4px;
font: 14pt &quot;Rockwell&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>0</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID_Fournisseur">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>951</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);border:2px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_accueilgerat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Accueil</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>20</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/R (1).png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 300 17pt &quot;Bahnschrift Light&quot;;</string>
   </property>
   <property name="text">
    <string>Adresse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gerer_fournisseur">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Gérer Les fournisseurs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>20</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid black;color:white;border-radius:10px;
font: 900 12pt &quot;Arial Black&quot;;
background-color: rgb(255, 114, 116);</string>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>445</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 300 17pt &quot;Bahnschrift Light&quot;;</string>
   </property>
   <property name="text">
    <string>Code Médicament</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 300 17pt &quot;Bahnschrift Light&quot;;</string>
   </property>
   <property name="text">
    <string>ID Fournisseur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Code_Med">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>450</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gerer_rupture">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Gérer Les ruptures</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 300 17pt &quot;Bahnschrift Light&quot;;</string>
   </property>
   <property name="text">
    <string>Numéro De Téléphone</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/devenir-pharmacien-921x1024.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>220</y>
     <width>201</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/42453.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gere_vente">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(204, 204, 204);;font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Gérer les ventes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);font-size:20px;
font: 20pt &quot;Segoe Script&quot;;border:2px solid black;
color: rgb(12, 255, 125);</string>
   </property>
   <property name="text">
    <string>PHARMACIE</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_addresse_fournisseur</zorder>
  <zorder>pushButton_gerer_stock</zorder>
  <zorder>pushButton_Supprimer_Fournisseur</zorder>
  <zorder>lineEdit_Numero_Telephone</zorder>
  <zorder>label_6</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEdit_Nom_fournisseur</zorder>
  <zorder>name_user</zorder>
  <zorder>pushButton_gerer_rupture_2</zorder>
  <zorder>ButtonProfil</zorder>
  <zorder>line_4</zorder>
  <zorder>label_10</zorder>
  <zorder>label_5</zorder>
  <zorder>line_2</zorder>
  <zorder>label_12</zorder>
  <zorder>pushButton_Precedent</zorder>
  <zorder>label_date</zorder>
  <zorder>line</zorder>
  <zorder>lineEdit_ID_Fournisseur</zorder>
  <zorder>pushButton_accueilgerat</zorder>
  <zorder>label_7</zorder>
  <zorder>label_11</zorder>
  <zorder>pushButton_gerer_fournisseur</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>label_14</zorder>
  <zorder>label_9</zorder>
  <zorder>lineEdit_Code_Med</zorder>
  <zorder>pushButton_gerer_rupture</zorder>
  <zorder>label_13</zorder>
  <zorder>label_8</zorder>
  <zorder>label_16</zorder>
  <zorder>pushButton_gere_vente</zorder>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
